app/services/twilio_service.rb

```diff
@@ -35,6 +35,21 @@ def from_number
     "+1#{account['number']}"
   end
 
+  def valid_number?(to_number)
+    to_number = format_phone_number(to_number)
+    begin
+      response = client.lookups.phone_numbers(to_number).fetch
+      response.phone_number
+      return true
+    rescue => e
```

Comment:
is there a particular Twilio error we can rescue instead of `StandardError` like this?